### PR TITLE
setup initial config

### PR DESCRIPTION
**Pull request by danmakenoise**
This is the initial implementation of our eslint config!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "eslint-config-inturn",
+  "version": "0.1.0",
+  "description": "an eslint config used by the development team at inturn",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (orrybaram) [inline package.json]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+
+2. If you haven't already, install `install-peerdeps` globally.
+```bash
+npm install -g install-peerdeps
+```
+
+3. Install the peer dependencies of `eslint-config-inturn`.
+```bash
+install-peerdeps --dev eslint-config-inturn
+```
```
Comment (orrybaram) [inline README.md:21]: `npx install-peerdeps --dev eslint-config-inturn` instead of installing it.

**Review by orrybaram — COMMENTED** [review]
Looks great one minor comment

File: README.md (hunk 2 above)
Comment (benox3) [reply README.md:21]: Personally, I don't think it's necessary to install `install-peerdeps`, I would suggest just a gentle reminder to not forget to install peer deps and let the user decide how they would like to install them. Not a big deal either way.

**Review by benox3 — APPROVED** [review]
LGTM

**Pull request merged**